src/containers/forms/OrganizationForm/tests/index.test.tsx

```diff
@@ -54,6 +51,7 @@ describe('containers/forms/Teamform', () => {
   it('form validation works', async () => {
     const wrapper = mount(<TeamForm />);
     wrapper.find('form').simulate('submit');
+    // tslint:disable-next-line
     await new Promise(resolve => setImmediate(resolve));
```

Comment:
What's the rule being violated by this line?

Reply:
Seems like we should just `.catch(e => throw e)` instead of ignoring the line

Reply:
I added a fix for this

Reply:
Interesting, how does adding `<any>` to `new Promise<any>(...)` stop the line from breaking the rule?

Reply:
@cKellyDesign  It was because this is the rule being violated https://palantir.github.io/tslint/rules/no-inferred-empty-object-type/, more info here: https://stackoverflow.com/questions/53570129/generic-type-on-typescript-class-can-be-omitted